content/09.overfitting.md

```diff
@@ -1,33 +1,36 @@
+![A visual example of overfitting and failure to generalize. While a high-degree polynomial achieves high accuracy on its training data, it performs poorly on data with specificities that have not been seen before. That is, the model has learned the training dataset specifically rather than learning a generalizable pattern that represents data of this type. In contrast, a simple linear regression works well on both datasets. The greater representational capacity of the polynomial is analogous to using a larger or deeper neural network.](images/overfitting.png){#fig:overfitting-fig}
 
-To evaluate deep supervised learning models, they should be trained, tuned, and tested on non-overlapping datasets.
-The data used for testing should be locked and only used one-time for evaluating the final model after all tuning steps are completed.
-Using a test set more than once will lead to biased estimates of the generalization performance [@arxiv:1811.12808; @doi:10.1162/089976698300017197].
-While transformation and normalization procedures need to be applied equally to all datasets, the parameters required for such procedures (for example, quantile normalization, a common standardization method when analyzing gene-expression data) should only be derived from training data, not tuning and test data, to keep the latter two independent.
-Additionally, many conventional metrics for classification (e.g. area under the receiver operating characteristic curve or AUROC) have limited utility in cases of extreme class imbalance [@pmid:25738806].
-Model performance should be evaluated with a carefully picked panel of relevant metrics that make minimal assumptions about the composition of the testing data [@doi:10.1021/acs.molpharmaceut.7b00578], with particular consideration given to metrics that are most directly applicable to the task at hand.
-
-The simplest way to combat overfitting is to detect it.
-This can be done by splitting the dataset into three parts: a training set, a tuning set (also commonly called a validation set in the machine learning literature), and a test set.
-By exposing the model solely to the training data during fitting, a researcher can use the model's performance on the unseen test data to measure the amount of overfitting.
+In general, one of the most effective ways to combat overfitting is to detect it in the first place.
+One way to do this is to split the main dataset being worked on into three parts: a training set, a tuning set (also commonly called a validation set in the machine learning literature), and a test set.
+These three partitions allow for the potentially iterative give and take between model learning (on the training set) and updating (on the tuning set) to happen independently of the final model evaluation (on the test set).
```

Comment:
Not sure about the term "give and take" here. Can we just say "These three partitions allow us to potentially iterate between model learning (on the training set) and updating (on the tuning set) to happen independently of the final model evaluation (on the test set)."

Reply:
```suggestion
These three partitions allow us to optimize models by iterating between model learning (on the training set) and hyperparameter evaluation (on the tuning set) without affecting the final model assessment on the test set.
```